src/style.css

```
@@ -0,0 +1,32 @@
+html,
+
+body,
+div {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url("images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:nth-child(n+2) {
+  margin-left: 4px;
+}
+
+.stars--5 .stars__star,
+.stars--4 .stars__star:not(:nth-child(n+5)),
```

You can simplify it a bit.
```suggestion
.stars--4 .stars__star:nth-child(-n+5),
```